p2p/host/basic/basic_host.go

```diff
@@ -148,6 +148,9 @@ type HostOpts struct {
 	// DisableSignedPeerRecord disables the generation of Signed Peer Records on this host.
 	DisableSignedPeerRecord bool
 
+	// EventBus must be set.
+	EventBus event.Bus
```

Comment:
This is annoying. `NewHost` will panic if the `EventBus` is not set.
Should we make the `event.Bus` a function parameter of `NewHost`?

Reply:
that sounds like the right thing to do